config.py

```diff
@@ -98,4 +103,4 @@ def recursively_load_dict_contents_from_group(self, h5file, path):
                 ans[key] = item.value
             elif isinstance(item, h5py._hl.group.Group):
                 ans[key] = self.recursively_load_dict_contents_from_group(h5file, path + key + '/')
-        return ans
\ No newline at end of file
+        return ans
```

Comment:
Just a minor suggestion here, we can either change display() to __repr__() or we could use the __dataclass__ in python3.7 to let it take care of these tasks (operator overload, etc.).